packages/block-editor/src/components/block-draggable/content.scss

```diff
@@ -18,3 +18,7 @@
 		content: none !important;
 	}
 }
+
+img:not([draggable]) {
```

Comment:
What is this about?

Reply:
I'll comment, but it's because images are by default draggable (even without the html attribute), so we want to prevent that so that you drag the whole image block instead.

Reply:
Do we want to be more specific though, like target only images that are block wrappers or things like that or do we want this for all images.

If it's for all images, maybe we can check whether it's the only html element with this behavior and whether we should have the same thing for other html elements.

Reply:
Looked for another solution but this seems to be the only way. So I changed it to only affect images in blocks. Also there doesn't seem to be any other element like this. Tested videos for example and they don't drag.

Reply:
Well, it turns out SVGs in Firefox were also draggable: https://github.com/WordPress/gutenberg/pull/67305#issuecomment-2505484832